wiki/adr-002-base-path-and-health-routes.md

```
@@ -38,16 +45,20 @@ We will implement configurable base paths and health routes through:
-    ReadyRoute  string `koanf:"ready_route"`
+    Path PathConfig `koanf:"path"`
+}
+
+type PathConfig struct {
+    Base   string `koanf:"base"`
+    Health string `koanf:"health"`
+    Ready  string `koanf:"ready"`
 }
 ```
 
 ### RouteRegistrar Interface
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)